src/stirling/source_connectors/socket_tracer/bcc_bpf/BUILD.bazel

```diff
@@ -70,7 +70,6 @@ pl_cc_test(
         "ENABLE_DNS_TRACING=true",
         "ENABLE_REDIS_TRACING=true",
         "ENABLE_NATS_TRACING=true",
-        "ENABLE_MUX_TRACING=true",
```

Comment:
Just FYI: This was fixed by @yzhao1012 previously, and just got merged. So you'll have to rebase.

Reply:
Funny timing :). I'll rebase and pull that in.

Reply:
Actually his change missed this file's duplication. So this needs to stay (see line 66 above)